contracts/mainnet/connectors/uniswapV3/main.sol

```diff
@@ -0,0 +1,203 @@
+pragma solidity ^0.7.6;
+pragma abicoder v2;
+
+/**
+ * @title Uniswap v3.
+ * @dev Decentralized Exchange.
+ */
+
+import {TokenInterface} from "../../common/interfaces.sol";
+import {Helpers} from "./helpers.sol";
+import {Events} from "./events.sol";
+
+abstract contract UniswapResolver is Helpers, Events {
+    /**
+     * @dev Mint New Position
+     * @param params: parameter for mint.
+     * @param getId: ID to retrieve amtA.
+     * @param setId: ID stores the amount of LP token.
+     */
+    function mint(
+        MintParams memory params,
+        uint256 getId,
+        uint256 setId
+    )
+        external
+        payable
+        returns (string memory _eventName, bytes memory _eventParam)
+    {
+        params.amtA = getUint(getId, params.amtA);
```

Comment:
@cryptoDev222 Since we have amtA and amtB we need to have two getIds

Reply:
Okay